### PR TITLE
Bitcoin backports #13756 #15770 #17730 #17750 #17752 #17826 #17857 #15886

### PR DESCRIPTION
**Pull request by knst**
- By default delay for dialogs is set to 0 instead of 3 seconds - are is it expected value in dash?
In bitcoin it was changed in this pull-request: bitcoin#9697
 - Backport bitcoin#13756 introduces a new feature "avoid_reuse" in wallet.
 - SendConfirmationDialog has now extra details (see a screenshot)
image

### CODE REVIEW

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
pls see https://github.com/UdjinM6/dash/commits/pr4887

**Comment by knst** [comment]
> pls see https://github.com/UdjinM6/dash/commits/pr4887

agree with all 3. Picked-up them.

Should I add in functional test a case of `sendtoaddress` with all 8 arguments that is not tested now?

**Review by UdjinM6 — APPROVED** [review]
ACK

File: depends/packages/qt.mk [hunk 1]
```diff
@@ -47,16 +47,15 @@ $(package)_config_opts += -no-ico
 $(package)_config_opts += -no-iconv
 $(package)_config_opts += -no-kms
 $(package)_config_opts += -no-linuxfb
-#$(package)_config_opts += -no-libjpeg
+$(package)_config_opts += -no-libjpeg
```
Comment (PastaPastaPasta) [inline depends/packages/qt.mk:50]: nit: why is 6e5440d505d351e95266e322ee73aff7fa0e72b4 not squashed into previous?

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Pull request merged**